src/main/kotlin/day16/exercise/datasource/image/ImageDataSourceImpl.kt

```
@@ -0,0 +1,34 @@
+    }
+
+    override suspend fun saveImage(bytes: ByteArray, path: String) = withContext(Dispatchers.IO) {
+        try {
+            val file = File(path)
+
+            // 파일이 이미 존재하면 덮어쓰기
+            FileOutputStream(file).use { outputStream ->
+                outputStream.write(bytes)
+            }
+
+        } catch (e: IOException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)